src/scripts/visualizations/line.js

```diff
@@ -139,7 +139,7 @@
         }
 
         function renderMarkers() {
-            var animationDelay = options.line.marker.animationDelay;
+            var animationDelay = (options.line.marker.animationDelay == null) ? duration : options.line.marker.animationDelay;
```

Comment:
good point. I think jshint doesn't complain because of the `/* jshint eqnull: true */` (line 68) placed before the wrapper render function inside which renderMarkers exists.

I used `==` as a convenient way to check for both undefined and null values - I got the idea from the comments in lines 384-385 of contour-utils.js (originally by Jaime, I believe).